app/views/index.scala.html

```diff
@@ -206,12 +206,12 @@
             </div>
             <section class="collaborator-images-container">
                 <div class="collaborator-image">
-                    <a href="https://makeabilitylab.cs.washington.edu/">
+                    <a id="makeabilitylab-img-link" href="https://makeabilitylab.cs.washington.edu/">
                         <img src='@routes.Assets.at("assets/MakeabilityLogo.png")' alt='Makeability_Logo'>
                     </a>
                 </div>
                 <div class="collaborator-image">
-                    <a href="https://ligapeatonal.org/">
+                    <a id="ligapeatonal-img-link" href="https://ligapeatonal.org/">
```

Comment:
There is also a link below that only shows up in Columbus, OH that should have an ID and get logged when clicked.